packages/interface/src/components/main-dashboard-button/index.js

```diff
@@ -6,25 +6,20 @@ import {
 	createSlotFill,
 } from '@wordpress/components';
 
-/**
- * Internal dependencies
- */
-import FullscreenModeClose from '../fullscreen-mode-close';
-
-const name = '__experimentalSiteEditorMainDashboardButton';
+const name = '__experimentalMainDashboardButton';
 
 const { Fill, Slot } = createSlotFill( name );
 
 const MainDashboardButton = Fill;
 MainDashboardButton.Slot = Slot;
 MainDashboardButton.slotName = name;
 
-export const CloseButton = () => {
+export const CloseButtonSlot = ( { children } ) => {
```

Comment:
Why not make this `MainDashboardButton.Slot`? Is there any place where we would just use the bare `MainDashboardButton.Slot`?

Reply:
Now that you mentioned it - no, we probably don't. :) I refactored this in 7d1b948db35d06cbd9bac92ebb81a3759ad34d08. I had to separate out the assignment though because the linter was complaining otherwise:

`React Hook "useSlot" is called in function "MainDashboardButton.Slot" that is neither a React function component nor a custom React Hook function`

Likely because it wasn't detected as valid component name for some reason. 🤔